src/main/scala/org/broadinstitute/dsde/workbench/sam/model/SamModel.scala

```diff
@@ -38,7 +38,7 @@ object SamJsonSupport {
 
   implicit val ResourceIdFormat = ValueObjectFormat(ResourceId.apply)
 
-  implicit val ResourceIdentityFormat = jsonFormat2(FullyQualifiedResourceId.apply)
+  implicit val FullyQualifiedResourceIdFormat = jsonFormat2(FullyQualifiedResourceId.apply)
```

Comment:
Unrelated to rest of PR, just noticed that this json formatter was named inconsistently